packaging/debian/centreon-engine.postinst

```diff
@@ -27,7 +27,6 @@ if [ "$1" = "configure" ] ; then
         /var/log/centreon-engine
     chmod -vR g+w \
         /etc/centreon-engine \
-        /var/lib/centreon-engine \
```

Comment:
nfpm is not used on dev-23.04.x, then I am not sure /var/lib/centreon-engine has the good rights
You may need to do `chmod -R g+w /var/lib/centreon-engine` only when it is a fresh install (`[ "$1" = "configure" ] && [ -z "$2" ]`)